app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.kt

```diff
@@ -83,11 +88,23 @@ class AddWheelchairAccessBusiness(o: OverpassMapDataAndGeometryApi) : SimpleOver
     override val icon = R.drawable.ic_quest_wheelchair_shop
     override val defaultDisabledMessage = R.string.default_disabled_msg_go_inside
 
-    override fun getTitle(tags: Map<String, String>) = R.string.quest_wheelchairAccess_name_title
+    override fun getTitle(tags: Map<String, String>) = 
+        if (hasFeatureName(tags) && !tags.containsKey("brand"))
```

Comment:
Why the check for brand? Don't you want _"What are the opening hours of McDonalds (fast food restaurant)?"_ Why not?

Reply:
see https://github.com/westnordost/StreetComplete/pull/2012#issuecomment-675608159
What I actually get is "What are the opening hours of McDonalds (`brand name in chinese`)?"
If there is a brand, then it is shown instead of type. Usually in English, sometimes in other languages.
And I have no clue where this comes from...

Reply:
Okay, I also know why, but why not? It is in Chinese if you are in China, right?

Reply:
No, I had it in Austria and Czechia, with the language set to UK English... and since I feel completely unable to solve this, I decided to take the easy way out :/

Reply:
Okay, that must be a bug iin the Feature Dictionary

Reply:
Ok I released a new version of the feature dictionary. 

https://github.com/westnordost/osmfeatures/blob/master/CHANGELOG.md

Would you try it out (update dependency in the build.gradle - 1.2)

Reply:
The code you should use is

```kotlin
   override fun getTitle(tags: Map<String, String>) = 
// if the feature can be named (generally), use the "Is NiceHaircut (barber shop) wheelchair accessible?" string 
        if (hasFeatureName(tags))
            R.string.quest_accepts_cash_type_title
// otherwise "Is NiceHaircut wheelchair accessible?" string 
        else
            R.string.quest_accepts_cash_title


    override fun getTitleArgs(tags: Map<String, String>, featureName: Lazy<String?>): Array<String> {
        val name = tags["name"] ?: tags["brand"]
// same here -> can it be named generally? Then supply both name and generic feature name
        if (featureName.value != null)
            arrayOf(name, featureName.value.toString())
// otherwise just supply the name
        else
            arrayOf(name)
    }

```
The comments are just for you to understand, please don't also commit them

Reply:
(Nevermind my deleted comment, it is fine the way it is)

Reply:
Works nicely, now it shows the English brand name.
However, I still think using brand instead of type can look strange (e.g. *"What are the opening hours of McDonald's (McDonald's)?"* ). So I would like to keep excluding brands from being shown.

Reply:
Ok, I would have thought that your `hasFeatureName(tags) && !tags.containsKey("brand")` check led to that in this case, the brand is not shown?

Reply:
With `if (hasFeatureName(tags) && !tags.containsKey("brand"))` it works like before, showing *"What are the opening hours of McDonald's* without brand or type.
But here the updated feature dictionary does not matter, which is why I initially tested with `if (hasFeatureName(tags))` (without the exclusion of brands). Then brands are still shown instead of type, but at least in English.